tests/garage/experiment/test_snapshot.py

```diff
@@ -5,7 +5,7 @@
 import joblib
```

Comment:
can you rename this test_snapshotter_integration.py? this is an integration test and it's kind of confusing when there are unit tests in test_snapshotter